.circleci/config.yml

```diff
@@ -19,10 +25,7 @@ jobs:
       - run: flake8 --config tools/.flake8.ini *.py **/**.py **/**/*.py
       - run: flow-coverage-report --config flow-coverage.json
       - run: flow check
-
-      # Build project
-      - run: cp config.conf.example config.conf
-      - run: make -j8 || true
+      - run: grunt qunit
```

Comment:
`make test` never gets run?

Reply:
@sushain97 did you want `make` to not create test files and `make test` to create test files?

Reply:
Seems sensical?

Reply:
In other words, currently `make test` works to create test files. However, `make` will always build all files for you including test files.

Reply:
@sushain97 Ahh oops, I realized what I did wrong :D